spec/features/feature_flags_spec.rb

```diff
@@ -45,19 +45,33 @@
       page.driver.browser.basic_authorize("flags", "password")
       visit feature_flags_path
       expect(page).to have_content "sentryNo"
-      expect(FeatureFlags.enabled?(:sentry)).to eq false
+      expect(FeatureFlags.enabled?(:sentry, without_session_data: true)).to eq false
 
       visit edit_feature_flag_path("sentry")
       choose "Yes"
       click_on "Save and continue"
       expect(page).to have_content "sentryYes"
-      expect(FeatureFlags.enabled?(:sentry)).to eq true
+      expect(FeatureFlags.enabled?(:sentry, without_session_data: true)).to eq true
 
       visit edit_feature_flag_path("sentry")
       choose "No"
       click_on "Save and continue"
       expect(page).to have_content "sentryNo"
-      expect(FeatureFlags.enabled?(:sentry)).to eq false
+      expect(FeatureFlags.enabled?(:sentry, without_session_data: true)).to eq false
+    end
+  end
+
+  context "when setting feature flags in the session" do
```

Comment:
wasn't sure where to put this test so ended up putting it in this file - also I'm not too sure about creating the example flag purely for testing. But seems to be the safest way to get this test passing without having to depend on temporary flags. It means though that there will always be a `feature_flags => { example => false }` on our session data